### PR TITLE
Allow developer to disable logging from mirage adapter

### PR DESCRIPTION
**Pull request by TomiHiltunen**
- Adds boolean parameter `logging` to Mirage adapter.
- The parameter can be used to control logging from within adapter.

### CODE REVIEW

File: packages/oats-mirage-adapter/index.ts [hunk 1]
```diff
@@ -45,8 +45,12 @@ async function handleFormData(value: FormData) {
 
 function adapter<Registry extends mirageTypes.AnyRegistry, RequestContext>(
   mirageServer: mirage.Server<Registry>,
-  requestContextCreator: (schema: Schema<Registry>, request: mirage.Request) => RequestContext
+  requestContextCreator: (schema: Schema<Registry>, request: mirage.Request) => RequestContext,
+  logging: boolean = true
```
[the remark this answers is not in the document]
Comment (sakari) [reply packages/oats-mirage-adapter/index.ts:49]: that would be nice

[the remark this answers is not in the document]
Comment (TomiHiltunen) [reply packages/oats-mirage-adapter/index.ts:49]: That's fine, I'll change it around.

[the remark this answers is not in the document]
Comment (sakari) [reply packages/oats-mirage-adapter/index.ts]: now you made all the params into an object -- which means we would get to write a changelog and bump big versions. This seems uncalled for.

[the remark this answers is not in the document]
Comment (TomiHiltunen) [reply packages/oats-mirage-adapter/index.ts]: Fine, I'll change it up again.

File: packages/oats-mirage-adapter/index.ts [hunk 2]
```diff
@@ -118,6 +121,11 @@ export function bind<
   handler: runtime.server.HandlerFactory<Spec>;
   spec: Spec;
   requestContextCreator?: (schema: Schema<Registry<Models, Factories>>) => RequestContext;
+  logging?: boolean;
```
Comment (JoonasSa) [inline packages/oats-mirage-adapter/index.ts:124]: I think this parameter should also be:

```
opts?: {
    logging?: boolean;
}
```

**Review by sakari — APPROVED** [review]

**Pull request merged**